### PR TITLE
Linting setup

### PR DESCRIPTION
**Pull request by marcelomorgado**
#29 
Should I create a npm `prettier` script?

### CODE REVIEW

**Comment by pcowgill** [comment]
@marcelomorgado Yeah, `npm run lint` would be a good script to have in the package.json, and then maybe we should make `npm test` also run prettier before running the test suite.

**Comment by pcowgill** [comment]
@marcelomorgado Also let's make a single commit where we commit all the changes to the files that autoformatting introduced.

**Comment by pcowgill** [comment]
@marcelomorgado What text editor are you using? Prettier format on save in VS Code is magical.

**Comment by marcelomorgado** [comment]
Script `npm run lint` createad and `npm run test` updated.

**Comment by marcelomorgado** [comment]
I'm using atom w/ prettier auto-saving.

**Comment by marcelomorgado** [comment]
About formatting existing files, I've not done that yet because this branch is outdated.

File: packages/tasit-sdk/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
   },
   "scripts": {
     "prepublish": "npx babel src --out-dir lib --ignore **/*.test.js",
-    "test": "npx mocha src/index.test.js --require @babel/register"
+    "test": "npm run lint && npx mocha src/index.test.js --require @babel/register",
+		"lint": "npx prettier src/* --write"
```
Comment (pcowgill) [inline packages/tasit-sdk/package.json]:
```suggestion
    "lint": "npx prettier src/* --write"
```

**Comment by pcowgill** [comment]
Ah yeah, cool, Atom configured like that works too!

Yeah, these configs are fine but I'm also open to changes. And if we change anything we should do it relatively soon, but it doesn't need to be right away. We can make a commit that reformats the project along with any changes.

File: packages/tasit-sdk/package.json (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-sdk/package.json]: @marcelomorgado Once we make this change we can merge this PR.

**Pull request merged**